README.md

```diff
@@ -133,6 +133,7 @@ In addition to the standard connection parameters the driver supports a number o
 | escapeSyntaxCallMode          | String  | select  | Specifies how JDBC escape call syntax is transformed into underlying SQL (CALL/SELECT), for invoking procedures or functions (requires server version >= 11), possible values: select, callIfNoReturn, call |
 | maxResultBuffer               | String  | null    | Specifies size of result buffer in bytes, which can't be exceeded during reading result set. Can be specified as particular size (i.e. "100", "200M" "2G") or as percent of max heap memory (i.e. "10p", "20pct", "50percent") |
 | gssEncMode                    | String  | prefer  | Controls the preference for using GSSAPI encryption for the connection,  values are disable, allow, prefer, and require |
+| lobVarlena                    | Boolean | false   | Treat Clobs as text and Blobs as bytea, instead of treating both as LargeObjects |
```

Comment:
Not a fan of the name of this parameter. clobAsText is certainly more direct

Reply:
I'm quite open to suggestion, but I think the name needs to reflect the fact that it applies to Blob/bytea as well as Clob/text.

Reply:
>to Blob/bytea as well as Clob/text.

I doubt it makes sense to tie different data types with a single configuration option. What if the user want to map blob=>lo, clob=>text?

Reply:
what would you like? two boolean settings? I'm OK with that, although I generally try not to proliferate settings.

Reply:
1) It has to be aligned with the existing settings
2) It has to be human-friendly
3) Backend [protocol ties encoding to the datatype](https://github.com/pgjdbc/pgjdbc/blob/master/backend_protocol_v4_wanted_features.md#binary-transfer-vs-exact-data-type), and you do even touch that in this PR: `connection.getLobVarlena() ? (connection.getStringVarcharFlag()`.